packages/storybook/stories/controlsUtils.ts

```diff
@@ -254,27 +254,11 @@ export const controlsToAdd = {
     name: 'Remote participants (comma separated)'
   },
   remoteParticipants: { control: 'object', defaultValue: defaultRemoteParticipants, name: 'Remote participants' },
-  screenShareLayoutWidth: {
-    control: {
-      type: 'range',
-      min: mediaGalleryWidthOptions.min,
-      max: mediaGalleryWidthOptions.max,
-      step: mediaGalleryWidthOptions.step
-    },
-    defaultValue: 850,
-    name: 'Width (px)'
-  },
-  screenShareSidePanelTileAspectRatio: {
-    control: 'select',
-    options: ['16:9', '3:2', '4:3', '5:9', '1:1'],
-    defaultValue: '16:9',
-    name: 'Side Panel Tile Aspect Ratio (Width:Height)'
-  },
-  screenShareSidePanelWidthRatio: {
+  screenShareExperience: {
```

Comment:
Is there documentation though that tells a developer how their video gallery gets into these state?

Reply:
Not yet. Will add a small section on screensharing in the docs.

Reply:
I don't think so. I'll check and add some if none

Reply:
@JamesBurnside 
adding snippets to story
![image](https://user-images.githubusercontent.com/82416644/142282339-7ac0ad63-dc8c-4890-b86a-218cf11aa2af.png)
![image](https://user-images.githubusercontent.com/82416644/142282267-c56416dc-ee6f-43e0-9104-c3182085bf60.png)